src/meta/mod.rs

```diff
@@ -56,8 +57,46 @@ impl Meta {
             println!("cannot access '{}': {}", path.display(), err);
             return Ok(meta);
         }
+		let mut content = Vec::new();
+
+		if list_hidden_files {
+			let mut current_meta;
+			let mut parent_meta;
+
+			let path_str = path.to_str().unwrap();
+
+			if path_str == ".." {
+				let temp_path = fs::canonicalize(&PathBuf::from("..")).unwrap();
```

Comment:
I think we might have to move this to `core.rs` line 70. Maybe something like 

```rust
match Meta::from_path_recursive(&fs::canonicalize(&path.to_path_buf()).unwrap(), depth, self.flags.display_all) {}
```

The issue is that we will not be handling for paths like `./../`. 

Once that is done, I think we should be able to completely remove the if else check here as we will always get the absolute path.

> There might also be other place in which we might be doing this check which will have to be taken care of later